### ISSUE TITLE
Prusa Connect Thumbnail Rotation Incorrect

### ISSUE THREAD

**Issue by shjmy** [prusa3d/Prusa-Firmware-ESP32-Cam#90]
<img width="330" alt="Screenshot 2024-11-27 at 1 26 27 PM" src="https://github.com/user-attachments/assets/76db4758-60bc-4d8d-b38b-0052786c8bc8">

<img width="283" alt="Screenshot 2024-11-27 at 1 29 48 PM" src="https://github.com/user-attachments/assets/a5bf9a2b-67f4-4950-ac67-57c6b88cc93d">

Some of the thumbnails like the ones in the "https://connect.prusa3d.com/printers/table?sortBy=state%2Cname" page and the printer/xxxxxxx/cameras view thumbnails don't seem to respond to the Prusacam setting rotations. However the main view in the printer/xxxxxxx/cameras seems to have the image in the correct orientation. I wasn't unsure if this should be solved in the firmware or in prusa connect so I am submitting it here. (Also I don't know where I should submit Prusa connect bugs)

**Comment by andgju** [comment]
Considering how complicated setting up the ESP32 cam is, changing the rotation in the ESP32 web interface is simple enough. The "user-friendly experience" is probably focused towards their new Buddy3D Camera.

My cameras are mounted upside down, so I need to flip the image. I do this in the ESP32 web interface, but there's no option to flip the view in the Prusa's mobile app. This means I have to pick between having the right orientation on my PC or in the app, which is pretty frustrating.

When I rotate the camera in ESP32 so the image in the app is correct, the preview image(smaller one) in Web Connect ends up upside down. However, the main image(bigger one) when I click on a printer is right side up. Basically, the preview and main image in Web Connect are 180° apart (see image below).

![image](https://github.com/user-attachments/assets/ba0f2289-d42b-4787-86c1-2fea4d10650d)

If the Prusacam settings have a rotation option, it should just work everywhere in Prusa Connect, including thumbnails. Also, adding a rotation setting in the mobile app would make things so much easier for users. That way, we wouldn't have to mess around with our camera setups to fix orientation problems.

**Comment by johnyHV** [comment]
Hello @andgju . Yes, this is issue on the Prusa Connect. But latest version Prusa Connect have support for image rotation (second icon from right side). so you don't have to rotate the image in the camera

**Comment by andgju** [comment]
yes @johnyHV That solves part of the problem. But there's still no way to rotate the camera in the app other than rotating it in the camera interface. So it's really a Prusa phone app problem. so maybe wrong to bring it up under prusa3d/Prusa-Firmware-ESP32-Cam
